File: docs/vsto/codesnippet/CSharp/WinFormInputCS/GetInputString.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WinFormInputCS
{
    public partial class GetInputString : Form
    {
        public GetInputString()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //<Snippet3>
            Globals.ThisWorkbook.WriteStringToCell(this.textBox1.Text);
            this.Dispose();
            //</Snippet3>
        }
    }
}

File: docs/vsto/codesnippet/VisualBasic/WinFormInput/GetInputString.vb
Public Class GetInputString

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles Button1.Click

        '<Snippet3>
        Globals.ThisWorkbook.WriteStringToCell(Me.TextBox1.Text)
        Me.Dispose()
        '</Snippet3>
    End Sub
End Class